library/log.inc

```diff
@@ -499,7 +499,14 @@ function create_tls_conn($host, $port, $localcert, $cafile) {
 function send_atna_audit_msg($user, $group, $event, $patient_id, $outcome, $comments)
 {
     /* If no ATNA repository server is configured, return */
-    if ($GLOBALS['atna_audit_host'] === null || $GLOBALS['atna_audit_host'] == "" || !($GLOBALS['enable_atna_audit'])) {
+    if (!array_key_exists('atna_audit_host', $GLOBALS)
+        || $GLOBALS['atna_audit_host'] === null
+        || $GLOBALS['atna_audit_host'] == ""
+        || !(
+            array_key_exists('enable_atna_audit', $GLOBALS)
+            && $GLOBALS['enable_atna_audit']
+        )
+    ) {
```

Comment:
This is an odd construct.  Seems it could be just:

    if (empty($GLOBALS['atna_audit_host']) || empty($GLOBALS['enable_atna_audit'] ) ) {